src/coord/src/catalog/builtin.rs

```diff
@@ -1541,7 +1541,40 @@ GROUP BY operator, worker",
     needs_logs: true,
 };
 
-// Next id BuiltinView: 5034
+pub const PG_CONSTRAINT: BuiltinView = BuiltinView {
+    name: "pg_constraint",
+    schema: PG_CATALOG_SCHEMA,
+    sql: "CREATE VIEW pg_constraint AS SELECT
+    NULL::pg_catalog.oid as oid,
+    NULL::pg_catalog.varchar(64) as conname,
```

Comment:
Why this instead of `text`? We use text for pg_enum.enumlabel which is also type name.

Reply:
Ah, I just went from the PG implementation details (63-byte VARCHAR), didn't notice we used `text` elsewhere.